templates/package-scripts/td-agent/rpm/post

```diff
@@ -82,3 +82,7 @@ echo "Configure <%= project_name %> to start, when booting up the OS..."
 if [ "$1" = "2" ]; then
   /sbin/service <%= project_name %> condrestart >/dev/null 2>&1 || :
 fi
+
+if [ <%= use_systemd %> ]; then
```

Comment:
Hmm, this will generate:
```
if [ true ]; then
  systemctl daemon-reload
fi
```
Why not use `<% if` here to just generate `systemctl daemon-reload`?

Reply:
May i know what exact syntax you are recommending here?..

Reply:
https://puppet.com/docs/puppet/5.5/lang_template_erb.html#erb-structure-and-syntax